direct_inelastic/MERLIN/calibration/create_MERInst_Files.py

```diff
@@ -248,8 +249,8 @@ def create_one2onemap(one2oneold,one2onenew):
     # The script stores all its results in Manit Default save directory and 
     
      # list of the known cycles with approximate  end dates:          
-    cycle_end_list = {date(2019,03,29):'184',date(2019,07,19):'191',date(2019,10,25):'192',\
-    date(2019,12,20):'193',date(2999,01,01):'future'}
+    cycle_end_list = {date(2019,0o3,29):'184',date(2019,0o7,19):'191',date(2019,10,25):'192',\
```

Comment:
It looks like there are additional changes to the `date()` call - was this intended?